rootfs/etc/motd

```diff
@@ -0,0 +1,10 @@
+
+
+=================================================
+Welcome to the SweetOps tutorials toolbox!
```

Comment:
```suggestion
Welcome to the SweetOps tutorial toolbox!
```

Reply:
@osterman I don't have the strongest opinion on this one, but if the image is going to be called `cloudposse/tutorials` then shouldn't we call it the tutorials toolbox? Maybe it does sound awkward though?

Reply:
Ya, same no strong feelings. We can leave it.